### PR TITLE
good tests on all components

### PR DESCRIPTION
**Pull request by Tmacintyre**

### CODE REVIEW

File: src/__test__/Footer.test.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  })
+  it("renders a src and alt", () => {
+    const div = document.createElement("div")
+    render(
+      <BrowserRouter>
+        <Footer />
+      </BrowserRouter>,
+        div
+    )
+  })
```
Comment (SunkissedQueen) [inline src/__test__/Footer.test.js]: look like you are missing an assertion

File: src/__test__/Header.test.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  })
+  it("renders a src and alt", () => {
+    const div = document.createElement("div")
+    render(
+      <BrowserRouter>
+        <Header />
+      </BrowserRouter>,
+        div
+    )
+  })
```
Comment (SunkissedQueen) [inline src/__test__/Header.test.js]: looks like the assertion is missing

File: src/__test__/Home.test.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  })
+  it("renders a src and alt", () => {
+    const div = document.createElement("div")
+    render(
+      <BrowserRouter>
+        <Home />
+      </BrowserRouter>,
+        div
+    )
+  })
```
Comment (SunkissedQueen) [inline src/__test__/Home.test.js]: same as other comments

File: src/__test__/NotFound.test.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  })
+  it("renders a src and alt", () => {
+    const div = document.createElement("div")
+    render(
+      <BrowserRouter>
+        <NotFound />
+      </BrowserRouter>,
+        div
+    )
+  })
```
Comment (SunkissedQueen) [inline src/__test__/NotFound.test.js]: same as other comments

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
Looks like the test cases about src and alt attributes are missing the assertions. Please either remove the test cases or add applicable assertions. NOTE: src and alt attributes are used for images.

**Comment by scott198989** [comment]
I went ahead and deleted 
  it("renders a src and alt", () => {
    const div = document.createElement("div")
    render(
      <BrowserRouter>
        <Home />
      </BrowserRouter>,
        div
    )
  }) this portion of the suite was not needed for any components.

**Review by SunkissedQueen — APPROVED** [review]

**Pull request merged**